src/main/java/com/epam/spring/homework3/controller/CategoryController.java

```diff
@@ -0,0 +1,54 @@
+package com.epam.spring.homework3.controller;
+
+import com.epam.spring.homework3.dto.CategoryDto;
+import com.epam.spring.homework3.service.CategoryService;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@Slf4j
+@RestController
+@RequiredArgsConstructor
+public class CategoryController {
+
+    private final CategoryService categoryService;
+
+    @ResponseStatus(HttpStatus.OK)
+    @GetMapping("/categories")
```

Comment:
Keep one style of url, I would recommend using singular, like /category and change "/categories" to "/category/all". And also please use @RequestMapping("/category") instead of duplicate same url in each method.
Take a look at other controllers, I can notice the same issue